src/tests/end2end/models/ResNet50V2Nchw.cpp

```diff
@@ -17,17 +17,17 @@
 
 static const std::string kModelPath = WEBNN_END2END_TEST_MODEL_PATH;
 
-class ResNetNchwTests : public WebnnTest {
+class ResNet50V2NchwTests : public WebnnTest {
```

Comment:
Keep the default name `ResNetNchwTests` that represent 50 version

Reply:
Thanks, How about using ResNet50V2NchwTests to explicitly indicate that it is not 101 model since there is 101 model? @fujunwei